src/css/custom.css

```diff
@@ -6,25 +6,31 @@
 
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #2e8555;
-  --ifm-color-primary-dark: #29784c;
-  --ifm-color-primary-darker: #277148;
-  --ifm-color-primary-darkest: #205d3b;
-  --ifm-color-primary-light: #33925d;
-  --ifm-color-primary-lighter: #359962;
-  --ifm-color-primary-lightest: #3cad6e;
+
+  --ifm-navbar-link-hover-color: #6495fd;
+  --ifm-footer-link-hover-color: #6495fd;
+  --ifm-menu-color-active: #20609f;
+  --ifm-link-color:#20609f;
+  --ifm-breadcrumb-color-active: #4973e8;
+  --ifm-color-primary: #c350ba;
+  --ifm-color-primary-dark: #c350ba;
+  --ifm-color-primary-darker: #c350ba;
+  --ifm-color-primary-darkest: #c350ba;
+  --ifm-color-primary-light: #c350ba;
+  --ifm-color-primary-lighter: #c350ba;
+  --ifm-color-primary-lightest: #c350ba;
   --ifm-code-font-size: 95%;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
 }
 
 /* For readability concerns, you should choose a lighter palette in dark mode. */
 [data-theme='dark'] {
-  --ifm-color-primary: #25c2a0;
-  --ifm-color-primary-dark: #21af90;
-  --ifm-color-primary-darker: #1fa588;
-  --ifm-color-primary-darkest: #1a8870;
-  --ifm-color-primary-light: #29d5b0;
-  --ifm-color-primary-lighter: #32d8b4;
-  --ifm-color-primary-lightest: #4fddbf;
+  --ifm-color-primary: #d693e1;
+  --ifm-color-primary-dark: #c97fd6;
+  --ifm-color-primary-darker: #b56bca;
+  --ifm-color-primary-darkest: #a357be;
+  --ifm-color-primary-light: #e1b6ec;
+  --ifm-color-primary-lighter: #e8c3f0;
+  --ifm-color-primary-lightest: #f2dfff;
```

Comment:
These changes are also in another PR, #51 Can we remove those changes from this PR and focus on only one change i.e. to the `features.md` file?